lib/plugins/multipartBodyParser.js

```diff
@@ -45,6 +45,9 @@ function multipartBodyParser(options) {
     var override = opts.overrideParams;
 
     function parseMultipartBody(req, res, originalNext) {
+        // save original body on req.rawBody and req._body
+        req.rawBody = req._body = undefined;
```

Comment:
Not technically necessary, but wanted to be consistent about these two attributes always being available anytime bodyParser is used.